src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
+    <sky-docs-code-example
+      heading="Static tabs with add and close"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/static-add-close"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Dynamic tabs with add and close"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/dynamic-add-close"
+    ></sky-docs-code-example>
+  </sky-docs-code-examples>
+
   <sky-docs-design-guidelines>
-    See documentation for how to setup design guidelines.
+    
+      <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+        >
+        <p>
+          Use tabs when users require quick access to different slices of the same data.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use tabs to show difference slices of the same data."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/tabs/tabs-usage-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+          
+        </sky-docs-design-guideline>
+          
+        <sky-docs-design-guideline>
+            <p>
+                Use tabs when users need to complete different tasks on a page and each task includs a significant amount of data.
+            </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use tabs to group content for different tasks."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/tabs/tabs-usage-2.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+          
+        <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+        >
+        <p>
+          Don't use tabs when a page already has a set of tabs. Change the workflow instead of using multiple sets of tabs. 
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use multiple sets of tabs on a page."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/tabs/tabs-usage-3.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't combine vertical tabs and horizontal tabs."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/tabs/tabs-usage-4.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+          
+        </sky-docs-design-guideline>
+        
+        <sky-docs-design-guideline>
+        <p>
+            Don't use tabs when they are likely to overflow the available horizontal space. The collapsed behavior of tabs will accommodate cases where users add many tabs themselves, but it is not intended as the default experience with a tab set. Use <a href="https://developer.blackbaud.com/skyux/components/vertical-tabs">vertical tabs</a> instead.
```

Comment:
The rationale behind this guideline is more about the weakness of tabs-in-a-dropdown versus vertical tabs. "Unused space" isn't the real concern here - the inferior experience of collapsed tabs is. I did revise this to add a comment that the collapsed behavior accommodates small screen resolutions, since I made that exact example image further below in Behavior.